coordinator/migrations/2024-02-02-075927_add_polls_table/up.sql

```diff
@@ -0,0 +1,37 @@
+-- Your SQL goes here
+
+CREATE TYPE "Poll_Type_Type" AS ENUM ('SingleChoice');
+
+CREATE TABLE polls
+(
+    id                 SERIAL PRIMARY KEY       NOT NULL,
+    poll_type          "Poll_Type_Type"           NOT NULL,
+    question           TEXT                     NOT NULL,
+    active             BOOLEAN                  NOT NULL,
+    creation_timestamp timestamp WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP
```

Comment:
how about `created_at` ? its shorter and less cumbersome when writing sql queries 😅

Reply:
I always use autocomplete ;)